account_fiscal_position_type/demo/account_chart_template.xml

```diff
@@ -4,7 +4,7 @@
     @author Sylvain LE GAL (https://twitter.com/legalsylvain)
     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 -->
-<!-- pylint: disable=duplicate-xml-record-id -->
+<!-- oca-hooks:disable=duplicate-xml-record-id -->
```

Comment:
@etobella @moylop260 @sbidoul Pre-commit fails with duplicate-xml-record-id, even though it is being disable. What's missing?

Reply:
See https://github.com/OCA/stock-logistics-barcode/commit/b2f25c6f76c557f76b05c6220aa830dcf7dce25a#diff-1f0d8b9f155c6d525074173a530d149cf9a1732344141356c0138d142f8eabaeR8

The check name is different.

Reply:
Thank you @pedrobaeza , I corrected it, but it still fails. I may be missing something.